strawberry/exceptions.py

```diff
@@ -1,19 +1,50 @@
-# TODO: add links to docs
+from __future__ import annotations
 
+from enum import Enum
 from typing import List, Set, Union
 
 from graphql import GraphQLObjectType
 
 from strawberry.type import StrawberryType
 
 
+# TODO: add links to docs
```

Comment:
This TODO still needs to be resolved

Reply:
We should add a section in the docs that shows all these exceptions we can thrown?

Reply:
Yes :+1: Can we open an issue and link to the issue number here in the comment?

Reply:
Here there is :) #1298

Reply:
Excellent. In the case that I was ambiguous, I meant in the source comment too :slightly_smiling_face: